apps/hash-ai-worker-ts/src/activities/flow-activities/research-entities-action/coordinating-agent.ts

```diff
@@ -110,18 +111,15 @@ const generateSystemPromptPrefix = (params: {
     Don't start sub-tasks in parallel which duplicate or overlap, or where one will depend on the result of another (do it in sequence).
     For simpler research tasks you might not need sub-tasks.
 
-    Do not waste resources and time trying to produce a table, report or any other form of output that isn't the proposed entities with all their properties.
-    
-    ${questionsAndAnswers ? `You previously asked the user clarifying questions on the research brief provided below, and received the following answers:\n${questionsAndAnswers}` : ""}
-
-    The "complete" tool for completing the research task will only be available once you have submitted
-      proposed entities that satisfy the research prompt.
+    The "complete" tool for completing the research task will only be available once you have gathered facts.
```

Comment:
Technically the "complete" tool is only available once entities have been discovered, should this maybe say so instead?
```suggestion
    The "complete" tool for completing the research task will only be available once entities have been discovered.
```